src/UnityExtension/Assets/Editor/GitHub.Unity/ApplicationCache.cs

```diff
@@ -56,7 +56,6 @@ public IEnvironment Environment
                         unityVersion = Application.unityVersion;
                     }
                     environment.Initialize(unityVersion, extensionInstallPath.ToNPath(), unityApplication.ToNPath(), unityAssetsPath.ToNPath());
-                    environment.InitializeRepository(!String.IsNullOrEmpty(repositoryPath) ? repositoryPath.ToNPath() : null);
```

Comment:
The call that should be removed is the one in `DefaultEnvironment` and not this one